src/PrestaShopBundle/Form/Admin/Sell/Product/Pricing/PricingType.php

```diff
@@ -64,6 +73,10 @@ public function __construct(
      */
     public function buildForm(FormBuilderInterface $builder, array $options)
     {
+        $catalogPriceRuleLink = $this->legacyContext->getAdminLink('AdminSpecificPriceRule', true, ['updatespecific_price_rule' => '', 'id_specific_price_rule' => 'catalog_price_rule_id']);
+        /** Adding % to make link more unique */
```

Comment:
I guess the `getAdminLink` doesn't allow you to insert the `%catalog_price_rule_id%` parameter right away? If so please explain it in the comment, if not then this extra line is overkill

Reply:
Added comment

Reply:
Thanks